crates/primitives/src/snapshot/mod.rs

```diff
@@ -7,3 +7,6 @@ mod segment;
 pub use compression::Compression;
 pub use filters::{Filters, InclusionFilter, PerfectHashingFunction};
 pub use segment::{SegmentHeader, SnapshotSegment};
+
+/// Default snapshot block count.
+pub const SNAPSHOT_BLOCK_NUMBER_CHUNKS: u64 = 500_000;
```

Comment:
nit: name is a bit confusing, I think we don't have "chunks" terminology in snapshots

maybe something like
```suggestion
pub const BLOCKS_PER_SNAPSHOT: u64 = 500_000;
```